azuredevops_build_definition_deploy/README.md

```diff
@@ -1,3 +1,17 @@
+# azuredevops_build_definition_deploy
+
+Module that allows the creation of a pipeline dedicated to deploy terraform code
```

Comment:
this module is used for iac and for code deploy.
trigger is usual manual from azure portal